src/components/UserSettingsPage/UserSettingsPage.module.css

```diff
@@ -0,0 +1,16 @@
+.FormControlInput {
+    width: 100%;
+}
+
+.FormControlLabel {
+    padding: var(--gap-s);
+    padding-left: var(--gap-m);
+    min-width: 80px;
```

Comment:
But 100 on ProjectSettingsPage

FormControlLabel and FormControl overrides are same. It would be nice to manage it via prop. Not via copy/paste. Create issue about this in bricks, plz.